static/app/views/dashboards/editAccessSelector.tsx

```diff
@@ -135,6 +135,7 @@ function EditAccessSelector({dashboard, onChangeEditAccess}: EditAccessSelectorP
         maxVisibleAvatars={1}
         avatarSize={25}
         renderUsersFirst
+        tooltipOptions={{disabled: true}}
```

Comment:
just wanted to make sure (I'm not remembering correctly) was taking out the  avatars tooltip run by Ale or Vasudha?

Reply:
Yeah, I actually like our tooltips over the one that appears after a long hover. They show up because a `title` property is passed down to the avatar

<img width="271" alt="Screenshot 2024-11-21 at 10 11 52 AM" src="https://github.com/user-attachments/assets/04c53822-27e6-4887-b8d2-46b8c29192c3">

Possibly we could update `static/app/components/avatar/baseAvatar.tsx` to only pass along title if there is no tooltip, and give it `aria-label={title}` to ensure it's still got a11y compatibility

Reply:
i.e. something like this in that file

```js
  const avatarComponent = (
    <StyledBaseAvatar
      data-test-id={`${type}-avatar`}
      ref={forwardedRef}
      className={classNames('avatar', className)}
      round={!!round}
      suggested={!!suggested}
      style={{...sizeStyle, ...style}}
      title={hasTooltip ? undefined : title}
      aria-label={title}
      {...props}
    >
      {hasError ? backup : imageAvatar}
    </StyledBaseAvatar>
  );
```